docs/content/app-protect/configuration.md

```diff
@@ -203,3 +208,289 @@ spec:
+          name: VIOL_FILE_UPLOAD
+        - block: true
+          description: Illegal request content type
+          name: VIOL_URL_CONTENT_TYPE
+        - block: true
+          description: Illegal static parameter value
+          name: VIOL_PARAMETER_STATIC_VALUE
+        - block: true
+          description: Illegal parameter value length
+          name: VIOL_PARAMETER_VALUE_LENGTH
+        - block: true
+          description: Illegal parameter data type
+          name: VIOL_PARAMETER_DATA_TYPE
+        - block: true
+          description: Illegal parameter numeric value
+          name: VIOL_PARAMETER_NUMERIC_VALUE
+        - block: true
+          description: Parameter value does not comply with regular expression
+          name: VIOL_PARAMETER_VALUE_REGEXP
+        - block: true
+          description: Illegal URL
+          name: VIOL_URL
+        - block: true
+          description: Illegal parameter
+          name: VIOL_PARAMETER
+        - block: true
+          description: Illegal empty parameter value
+          name: VIOL_PARAMETER_EMPTY_VALUE
+        - block: true
+          description: Illegal repeated parameter name
+          name: VIOL_PARAMETER_REPEATED
+
+~~~
+
+Content of the referenced file `myapi.yaml`:
+
+~~~yaml
+openapi: 3.0.1
+info:
+  title: 'Primitive data types'
+  description: 'Primitive data types.'
+  version: '2.5.0'
+servers:
+  - url: http://localhost
+paths:
+  /query:
+    get:
+      tags:
+        - query_int_str
+      description: query_int_str
+      operationId: query_int_str
+      parameters:
+        - name: query_int
+          in: query
+          required: false
+          allowEmptyValue: false
+          schema:
+            type: integer
+        - name: query_str
+          in: query
+          required: false
+          allowEmptyValue: true
+          schema:
+            type: string           
+      responses:
+        200:
+          description: OK
+        404:
+          description: NotFound
+~~~
+
+In this case, the following request will trigger an `Illegal parameter data type` violation, as we expect to have an integer value in the `query_int` parameter:
+
+```
+http://localhost/query?query_int=abc
+```
+
+The request will be blocked.
+
+The `link` option is also available in the `openApiFileReference` property and is synonymous with the `open-api-files` property as seen in the App Protect policy example above.
+
+**Note**: `openApiFileReference` is not an array.
+
+
+## Configuration in NGINX Plus Ingress Controller using Virtual Server Resource
+In this example we deploy the NGINX Plus Ingress Controller with NGINX App Protect, a simple web application and then configure load balancing and WAF protection for that application using the VirtualServer resource.
+
+Refer to the below path to access these files :
+```
+https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v2.3.0/examples/custom-resources/waf/
+```
```

Comment:
Note: This example, and the files referenced, can be found [here](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/waf).